lib/ui/views/app_selector/app_selector_viewmodel.dart

```diff
@@ -69,6 +70,36 @@ class AppSelectorViewModel extends BaseViewModel {
     return true;
   }
 
+  Future<void> searchSuggestedVersionOnWeb(BuildContext context,
+      {required String packageName, required String appName, bool isInstalled = false,}) async {
+    final String search = FlutterI18n.translate(
+      context,
+      'appSelectorCard.search',
+    );
+    final String apk = FlutterI18n.translate(
+      context,
+      'appSelectorCard.apk',
+    );
+    final String version = FlutterI18n.translate(
+      context,
+      'appSelectorCard.version',
+    );
```

Comment:
You shouldnt translate these strings, for example translating "search" would likely break the search result